assembl/static2/js/app/utils/translations.js

```diff
@@ -214,6 +214,13 @@ const Translations = {
       thread: {
         messageTranslatedFrom: "Ce message a été traduit du texte %{language}.",
         messageOriginallyIn: "Ce message a été rédigé en %{language}.",
+        translateAllMessagesIn: "Je souhaite traduire tous les messages rédigés en %{language}.",
+        translateOnlyThisMessage: "Je souhaite traduire ce message.",
+        untranslateOnlyThisMessage: "Je souhaite afficher uniquement ce message dans sa langue originale",
+        untranslateAllMessagesIn: "Je souhaite afficher tous les messages rédigés en %{language} dans leur langue originale.",
+        chooseLanguagePh: "Veuillez choisir la langue",
+        translateCancel: "Annuler",
```

Comment:
You don't need to create a new key for this value. It already exists in the translations file. Check the `cancel` key.

Reply:
ok, I think I checked this but as it is not in a generic "path" in the translations, I thought we prefer to use different keys for each use in case we need to change the translation for a specific use (though for cancel/validate it is probably useless)

Reply:
done